tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -13,105 +13,105 @@
 # limitations under the License.
 # ==============================================================================
 import operator
-
 import unittest
-from functools import partial
 
 import numpy as np
 import torch
 from torch import nn
+
 import model_compression_toolkit as mct
-from model_compression_toolkit.core.common.hessian import HessianEstimationDistribution
 from model_compression_toolkit.core.common.mixed_precision.distance_weighting import MpDistanceWeighting
 from model_compression_toolkit.core.common.network_editors import NodeTypeFilter, NodeNameFilter
+from model_compression_toolkit.core.pytorch.pytorch_device_config import get_working_device
 from model_compression_toolkit.gptq.common.gptq_config import RoundingType
 from model_compression_toolkit.gptq.pytorch.gptq_loss import sample_layer_attention_loss
 from model_compression_toolkit.target_platform_capabilities import constants as C
+from model_compression_toolkit.target_platform_capabilities.target_platform import QuantizationMethod
 from model_compression_toolkit.trainable_infrastructure import TrainingMethod
+from tests.pytorch_tests.model_tests.feature_models.activation_16bit_test import Activation16BitTest, \
```

Comment:
Why are all the imports changes?

Reply:
I guess there were unused imports and I used the automatic tool to organize them, which changed the order. No real changes